libs/input-mapping/src/Reader.php

```diff
@@ -135,66 +129,19 @@ public static function getFiles(
         LoggerInterface $logger,
         InputFileStateList $fileStateList
     ): array {
-        $fileConfigurationRewritten = TagsRewriteHelper::rewriteFileTags(
+        $fileOptions = new InputFileOptions(
```

Comment:
Tady je strašlivej fuckup v tom, že 
Reader::downloadFiles volá nějaký file strategy a to pak zase zpětně volá tuhle zprasenou metodu Reader::getFiles, která se navíc ještě používá v OM.

Jako první krok jsem teda do ní hrábl a vytáhl tu logiku ven do InputFileOptions analogicky jako je to u tabulek. Zatím neměním ten interface, ať se to dá aspoň hypoteticky obsáhnout, ten interface změním v dalším PR a bude to major verze.